webapp/src/containers/ConsolePage/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface ConsolePageState {
+  isLoading: boolean;
+  result: any;
+  isError: string;
+  cliLog: any;
+}
```

Comment:
Generally, I think if types can be co-located within the same file it should done that way. Separating it, gives a false sense of separation of concerns as they are tightly related. 

What do you think?

Reply:
@fuxingloh I tried this approach first but some files have extremely long states. If you want to update the reducer/state, you need to access a very long file. Having this approach, opens up the possibility to create our own typings file. To me, I still prefer this style since it's cleaner to import from types than from reducer everytime.